packages/mobile/src/screens/edit-track-screen/screens/ReleaseDateScreen.tsx

```diff
@@ -0,0 +1,289 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import type { Nullable } from '@audius/common'
+import { remixSettingsActions, removeNullable } from '@audius/common'
+import { useField } from 'formik'
+import moment from 'moment'
+import { Dimensions, View } from 'react-native'
+import DateTimePickerModal from 'react-native-modal-datetime-picker'
+import { useDispatch } from 'react-redux'
+
+import { IconCalendarMonth, Text } from '@audius/harmony-native'
+import { Button } from 'app/components/core'
+import { TextField } from 'app/components/fields'
+import { HelpCallout } from 'app/components/help-callout/HelpCallout'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import type { ListSelectionData } from './ListSelectionScreen'
+import { ListSelectionScreen } from './ListSelectionScreen'
+
+export enum ReleaseDateType {
+  RELEASE_NOW = 'RELEASE_NOW',
+  SCHEDULED_RELEASE = 'SCHEDULED_RELEASE'
+}
+
+const { reset } = remixSettingsActions
+
+const screenWidth = Dimensions.get('screen').width
+
+const messages = {
+  screenTitle: 'Release Date',
+  done: 'Done',
+  futureReleaseHint:
+    'Your scheduled track will become live on Audius on the date and time you’ve chosen above in your time zone (CST).',
```

Comment:
CST should be the actual local time zone, right?

Reply:
oops yea good catch i will follow up with a local time zone